index.js

```diff
@@ -44,10 +44,16 @@ module.exports = (options = {}) => {
   // Log all unhandled rejections
   process.on('unhandledRejection', logger.error.bind(logger));
 
+  // Deprecate SENTRY_URL
+  if (process.env.SENTRY_URL && !process.env.SENTRY_DSN) {
+    process.env.SENTRY_DSN = process.env.SENTRY_URL;
+    console.warn('DEPRECATED: the `SENTRY_URL` key is now called `SENTRY_DSN`. Use of `SENTRY_URL` is deprecated and will be removed in 0.10.0');
```

Comment:
> …will be removed in 0.10.0.

We should probably adopt an official deprecation policy, but recently I've been assuming that deprecated features will be around for one full minor version (while probot is <1.0, and one full major version once 1.0 is released). So a feature deprecated in 0.9 will still work in 0.10 and is removed in 0.11. This ensures that, as long as you don't currently have any deprecation warnings, you can always update to the next minor version without anything breaking.